### ISSUE TITLE
Support for google scholar

### ISSUE THREAD

**Issue by JFrandon** [NLeSC/litstudy#75]
Hi, is there a plan to support google scholar?

**Comment by stijnh** [comment]
No. Unfortunately, Google Scholar does not offer an accessible web API that we can use. 

Scraping the website is very difficult (and possible not legal). There is also no way to export data from Google Scholar.
